tutor/src/screens/task/step/failure.js

```diff
@@ -17,45 +18,53 @@ const StyledFailure = styled(StepCard)`
 class Failure extends React.Component {
 
   static propTypes = {
-    task: PropTypes.shape({
-      id: idType,
-    }),
-    step: PropTypes.shape({
-      id: idType,
-    }),
+    task: PropTypes.instanceOf(Task).isRequired,
+    step: PropTypes.instanceOf(Step).isRequired,
   }
 
   componentDidMount() {
-    const { task } = this.props;
+    const { task, step } = this.props;
+    let errMsg = [];
     if (!task) {
-      return Raven.log('Failed to load assignment task');
+      errMsg.push('Failed to load assignment task');
     }
     if (task.api.hasErrors) {
-      return Raven.log(`Failed to load assignment, errors: ${titleize(task.api.errors)}`);
+      const { last: _, ...errors } = task.api.errors;
+      errMsg.push(`Failed to load assignment, errors: ${titleize(errors)}`);
     }
-
-    const { step } = this.props;
     if (!step) {
-      return Raven.log('Failed to load assignment step');
+      errMsg.push(`Failed to ${this.requestAction} assignment step`);
+    }
+
+    const { last: _, ...errors } = step.api.errors;
+    errMsg.push(`Failed to ${this.requestAction} assignment step id: ${step.id}, error: ${titleize(errors)}`);
```

Comment:
Ohhh so the errors might have happened when trying to save... interesting.

Reply:
yeah, but hopefully not since that's pretty bad.  But if it does we need to log it so we can pay extra attention.